ui/app/components/app/modals/new-account-modal/index.scss

```diff
@@ -0,0 +1,37 @@
+.new-account-modal {
```

Comment:
- [ ] These styles aren't getting applied for some reason, probably need to import this scss file in the appropriate parent